go.mod

```diff
@@ -37,20 +36,19 @@ require (
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
 	github.com/uber/tchannel-go v1.14.0
-	go.uber.org/atomic v1.5.1
+	go.uber.org/atomic v1.4.0
```

Comment:
This (and x/lint) downgrade is a bit odd.  Possibly it's not imported locally + transitive dependencies changed and this is the new minimum?
It's very likely safe / good to upgrade to latest.  Latest does make a breaking change though, disallowing comparison between atomic wrappers (which is a good thing, as that's racy behavior).